test/test_run.py

```diff
@@ -82,8 +81,8 @@ def _test_run(range_spec, branches, expected_commits):
         expected = set(['machine.json'])
         for commit in expected_commits:
             for psver in tools.DUMMY2_VERSIONS:
-                expected.add('{0}-{1}-py{2}-asv_dummy_test_package_1-asv_dummy_test_package_2{3}.json'.format(
-                    commit[:8], tool_name, pyver, psver))
+                expected.add(f'{commit[:8]}-{tool_name}-py{pyver}-asv_dummy_'
+                             f'test_package_1-asv_dummy_test_package_2{psver}')
```

Comment:
You areremoving the `.json` here.
```suggestion
                             f'test_package_1-asv_dummy_test_package_2{psver}.json')
```

Reply:
Interesting how i had missed it thank you